lib/services/devices/deviceService.js

```diff
@@ -256,7 +256,7 @@ function formatCommandsNgsi2(originalVector) {
             };
```

Comment:
Should we add something to CNR? Something about defaults was previously wrongly set or something like that...

Reply:
(I didn't realize about the [WIP]... sorry)